lib/resty/limit/count.lua

```diff
@@ -41,53 +41,41 @@ function _M.incoming(self, key, commit)
     local limit = self.limit
     local window = self.window
 
-    local remaining, ok, err
+    local count, ok, err
 
     if commit then
-        remaining, err = dict:incr(key, -1, limit)
-        if not remaining then
+        count, err = dict:incr(key, 1, 0, window)
```

Comment:
Referring to #21, the reason why this module used both `incr` and `expire` is that `incr` didn't support `init_ttl` yet, I think.
So if you use `init_ttl` here, L53-L59 are not necessary.

Reply:
Indeed. I was using latest stable OpenResty and this got in  1.13.6.2 RC1. That explains why it was failing for me locally.